reactioncommerce/admin
# chore: changing foundation to create-react-app with typescript

**_Changing foundation to `create-react-app` with typescript._**

Testing: 
1. Run `npm install` and `npm run start`.
2. Open localhost:3000 and you should see react-admin successfully rendering.
3. Run `npm run test`. A single test should pass.
4. Make a small change and add it, and attempt a commit message that should fail (ie `git commit -m"testing").
5. This bad commit message should be caught by the commit linter and should only pass with a correct message (ie `git commit -m"chore: testing commit message").

### Note: Linting still needs external testing. To do this, remove the first line of src/App.tsx and run `npm run lint`.
